go.mod

```diff
@@ -4,6 +4,6 @@ go 1.13
 
 require (
 	github.com/newrelic/newrelic-telemetry-sdk-go v0.2.0
-	go.opentelemetry.io/otel v0.2.0
-	google.golang.org/grpc v1.24.0
+	go.opentelemetry.io/otel v0.3.0
+	google.golang.org/grpc v1.27.1
```

Comment:
I assume we only require grpc for the status code stuff. Unfortunate that we'd need to pull it in here. But ultimately not a problem because opentelemetry itself requires it. 👍

Reply:
Yeah, that's all my understanding as well.